finality-provider/service/fp_instance.go

```diff
@@ -1016,3 +1016,83 @@ func (fp *FinalityProviderInstance) GetFinalityProviderSlashedOrJailedWithRetry(
 
 	return slashed, jailed, nil
 }
+
+// CommitPubRandTiming - helper struct used to capture times for benchmark
+type CommitPubRandTiming struct {
```

Comment:
can we move all the helper functionalities to a separate file, maybe `benchmark_helper.go`?